### ISSUE TITLE
Add methods in pysmt.test.TestCase

### ISSUE THREAD

**Issue by marcogario** [pysmt/pysmt#65]
In many tests, we build a formula and test whether it is sat/unsat/valid. We should make these tests an assertion in the custom pysmt TestCase:
```
  def assertValid(self, formula, message=None, logic=None)
  def assertSat(self, formula, message=None, logic=None)
  def assertUnsat(self, formula, message=None, logic=None)
```

**Comment by mikand** [comment]
Like!

**Comment by mikand** [comment]
Closed in #67
